gym_roboy/envs/tests/test_msj_env.py

```diff
@@ -99,5 +102,24 @@ def test_msj_env_reward_is_lower_with_joint_vel_penalty():
     assert reward_with_no_joint_vel_penalty > reward_with_joint_vel_penalty
 
 
+def test_msj_env_agent_gets_bonus_when_reaching_the_goal():
+
+    msj_env = MsjEnv(is_agent_getting_bonus_for_done=False)
+    obs = msj_env.reset()
+    current_joint_angle = obs[0:3]
```

Comment:
now we can access this differenty: `ms_env._last_state.joint_angle` That way we get rid of the magic number 3.

Reply:
done